app/jobs/event_notification_job.rb

```diff
@@ -0,0 +1,9 @@
+class EventNotificationJob < ApplicationJob
+  def self.perform
```

Comment:
I will add stories to implement a scheduling provider and a background processing queue provider.
We wont actually be running this stuff until we set up a staging environment

For now this only requires a rake task to manually run the job from the console. In the Development environment by default Active job will run jobs inline, until we implement a background processing queue